### DISCUSSION TITLE
Uptime monitoring

### DISCUSSION THREAD

**Discussion by JustanotheruserNick** [alexjustesen/speedtest-tracker#1705] [Ideas]
Hey! I saw [this](https://github.com/alexjustesen/speedtest-tracker/discussions/1367#discussioncomment-9104137) comment made back in April of this year (2024) in my little research I've done. It says that it'll be added sometime after v0.20.0 I am just wondering if you've since worked on it at all? I do like the idea of having an uptime and speedtest all in one container. 

I appreciate your work and use it all the time!

**Comment by alexjustesen** [comment]
Short answer, no. But it is still planned just lacking the capacity at the moment and I want to shore up the existing feature set before adding more.
